index.js

```diff
@@ -3,11 +3,11 @@
 var fs = require('fs');
 var pp = require('preprocess');
 
-function preprocessPreprocessor(config, helper) {
+function preprocessPreprocessor(args, config, helper) {
   return function(content, file, done) {
     config = config || {};
 
-    var options = config.options || {};
+    var options = config.options || args.options || {};
```

Comment:
Here I think it is better to make a merge of the options instead of overwrite than. You can use the ```helper.merge``` method.